### ISSUE TITLE
Replace pancake tooltips and aria-labels

### ISSUE THREAD

**Issue by aubundy** [Rug-Zombie/rug-zombie-uikit#9]
Here's a picture of a leftover pancake tooltip. This one appears whenever the mouse hovers over the profile picture. 

These should be replaced with relevant zombie tooltips. It would be good to replace any aria-labels at the same time. I can assign this to myself if that's good @rugzombie @nnamss 

<img width="65" alt="Screen Shot 2021-10-24 at 10 41 19 PM" src="https://user-images.githubusercontent.com/37122448/138631454-98f8bde6-6b66-4835-b329-292bd411d866.png">
